services/sources/publisher_backlist_service.py

```diff
@@ -1,10 +1,12 @@
 from datetime import datetime, timedelta, timezone
+import json
 import os
 import requests
-import json
 import urllib.parse
 from typing import Optional
-from model import Record, Work, Edition, Item, Link
+from model import Record, Work, Edition, Item
+from sqlalchemy.orm import joinedload
+from urllib.parse import urlparse
```

Comment:
probably possible to just change this instance to be the same access pattern as the other urllib.parse usages